### ISSUE TITLE
Please specify minimum and maximum tested version of Python.

### ISSUE THREAD

**Issue by MattGyverLee** [rmlockwood/FLExTrans#512]
When you install FLExTrans (and FlexTools), I see that it offers to install a specific version of Python. I'm sure that is sufficient for people who don't use Python for other things. Unfortunately, the suggested version has NEVER been the version that I'm currently using for other projects and I want to know if I can continue using the version I have.

The last workshop, I believe I used Python 3.9 though 3.8 was suggested. I've since replaced my hard drive and done a clean install.

Because FLExTools and FLExTrans default to the Windows Path and don't allow specifying the specific installation at runtime, I basically have to stop working on other projects and roll back to an old version of Python while dealing with FlexTrans, and then reset my path and logout/restart to continue. I can sometimes tweak the bat and VBS files to make it work one version higher, but this is trial and error. 

Preference 1: 
Please include in the release notes the minimum and maximum tested versions of Python. I assume that could be generated based on the requirements (cdfutils and FlexTools), which seem to be the sticking packages that don't update.
Preference 2 (requires code): 
Please allow me to choose a pre-existing python install if I have one that is known compatible and adjust the .bat and .vbs files accordingly.

**Comment by rmlockwood** [comment]
I've stated, in the install instructions, that the minimum version of Python needed is 3.11, but also that any higher version of Python will work and does not have to be installed if you have a higher version. The FLExTrans start file (FlexTools.vbs) has the command py -3.11 so that will take any higher version of Python and dumb it down to 3.11.
